### PR TITLE
Draft: Add SST option for security in federated execution

### PR DESCRIPTION
**Pull request by Jakio815**
This PR is related to lf-lang/reactor-c#143

## SST ##
SST (Secure Swarm Toolkit) is an open-source security tool to authenticate and authorize entities, using a local point of authorization **Auth**.

### CODE REVIEW

File: org.lflang/src/org/lflang/TargetConfig.java [hunk 1]
```diff
@@ -232,6 +232,11 @@ public class TargetConfig {
      */
     public boolean auth = false;
 
+    /**
+     * Pass the path of the sst configuration file.
```
Comment (edwardalee) [inline org.lflang/src/org/lflang/TargetConfig.java:236]:
```suggestion
     * The path of the SST configuration file.
```

File: org.lflang/src/org/lflang/TargetProperty.java [hunk 2]
```diff
@@ -66,6 +66,18 @@ public enum TargetProperty {
                 config.auth = ASTUtils.toBoolean(value);
             }),
 
+    /**
+     * Path of configuration file of SST. It allows including OpenSSL libraries 
+     * and use SST for authentication and authorization.
```
Comment (edwardalee) [inline org.lflang/src/org/lflang/TargetProperty.java:71]:
```suggestion
     * and using SST for authentication and authorization.
```

**Review by edwardalee — COMMENTED** [review]
Minor typos in comments with suggested fixes.

File: org.lflang/src/org/lflang/TargetProperty.java (hunk 2 above)
Comment (Jakio815) [reply org.lflang/src/org/lflang/TargetProperty.java:71]: Thanks for your comments. This PR is not finished yet, and I made it to check the changed files. Thanks for the changes. I'll request a review when I'm finished.😁

**Pull request closed without merging**